src/abstractdataframe/join.jl

```diff
@@ -280,7 +286,16 @@ join(name, job2, on = :ID => :identifier)
 function Base.join(df1::AbstractDataFrame,
                    df2::AbstractDataFrame;
                    on::Union{<:OnType, AbstractVector{<:OnType}} = Symbol[],
-                   kind::Symbol = :inner, makeunique::Bool=false)
+                   kind::Symbol = :inner, makeunique::Bool=false,
+                   indicator::Union{Nothing, Symbol} = nothing)
+
+    if !( indicator == nothing )
```

Comment:
typically comparison to `nothing` is done using `===` AFAIK.

Reply:
also spaces at `(` and `)` are not needed I guess

Reply:
@bkamins is there a PEP8 for Julia yet about this kind of spacing stuff?

Reply:
For both comments above this is just style. But if you grep Julia sources in Base you will see that this is a predominant style (to use `===` to compare to `nothing` and not to put spaces after `(` and before `)`). Anyway this is a minor issue.